chainer/link.py

```diff
@@ -714,27 +830,25 @@ def __len__(self):
         """Returns the number of children."""
         return len(self._children)
 
-    def add_link(self, link):
-        """Registers a child link to this chain.
+    def append(self, link):
+        """Registers a child link and adds it to the tail of the list.
+
+        This is equivalent to :meth:`add_link`. This method has been added to
+        emulate the ``list`` interface.
+
+        Args:
+            link (Link): The link object to be regsitered.
 
-        The registered link is saved and loaded on serialization and
-        deserialization, and involved in the optimization. The registered link
-        is called a child. The child link is accessible via :meth:`children`
-        generator, which returns a generator running through the children in
-        registered order.
+        """
+        self.add_link(link)
 
-        This method also sets the :attr:`~Link.name` attribute of the
-        registered link. If the given link already has the name attribute set,
-        then it raises an error.
```

Comment:
It seems so, if my understanding is correct.
<img width="2522" alt="2017-05-29 21 32 06" src="https://cloud.githubusercontent.com/assets/407052/26549924/589291ae-44b6-11e7-846c-fb41b90104d8.png">